Exception downloading Thread script to file.

Its not a RN 0.56 problem, rather actually putting the threads file in any Directory other than app root causes this. 

```javascript
const thread = new Thread('./services/contacts.js');
thread.onmessage = (message) => console.log(message);
```

```java
Exception in native call
                              java.lang.RuntimeException: Exception downloading thread script to file
                                                                               at com.reactlibrary.RNThreadModule.downloadScriptToFileSync(RNThreadModule.java:220)
                                                                               at com.reactlibrary.RNThreadModule.createDevBundleLoader(RNThreadModule.java:175)
                                                                               at com.reactlibrary.RNThreadModule.startThread(RNThreadModule.java:68)
                                                                               at java.lang.reflect.Method.invoke(Native Method)
                                                                               at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:372)
                                                                               at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:160)
                                                                               at com.facebook.react.bridge.queue.NativeRunnable.run(Native Method)
                                                                               at android.os.Handler.handleCallback(Handler.java:751)
                                                                               at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                               at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:29)
                                                                               at android.os.Looper.loop(Looper.java:154)
                                                                               at com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run(MessageQueueThreadImpl.java:192)
                                                                               at java.lang.Thread.run(Thread.java:761)
                                                                            Caused by: java.io.FileNotFoundException: /data/user/0/com.testproject/files/services/contacts (No such file or directory)
                                                                               at java.io.FileOutputStream.open(Native Method)
                                                                               at java.io.FileOutputStream.<init>(FileOutputStream.java:221)
                                                                               at java.io.FileOutputStream.<init>(FileOutputStream.java:169)
                                                                               at okio.Okio.sink(Okio.java:181)
                                                                               at com.reactlibrary.RNThreadModule.downloadScriptToFileSync(RNThreadModule.java:217)
                                                                               at com.reactlibrary.RNThreadModule.createDevBundleLoader(RNThreadModule.java:175) 
                                                                               at com.reactlibrary.RNThreadModule.startThread(RNThreadModule.java:68) 
                                                                               at java.lang.reflect.Method.invoke(Native Method) 
                                                                               at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:372) 
                                                                               at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:160) 
                                                                               at com.facebook.react.bridge.queue.NativeRunnable.run(Native Method) 
                                                                               at android.os.Handler.handleCallback(Handler.java:751) 
                                                                               at android.os.Handler.dispatchMessage(Handler.java:95) 
                                                                               at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:29) 
                                                                               at android.os.Looper.loop(Looper.java:154) 
                                                                               at com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run(MessageQueueThreadImpl.java:192) 
                                                                               at java.lang.Thread.run(Thread.java:761) ```

--- a/android/src/main/java/com/reactlibrary/RNThreadModule.java
+++ b/android/src/main/java/com/reactlibrary/RNThreadModule.java
@@ -169,7 +169,11 @@ public class RNThreadModule extends ReactContextBaseJavaModule implements Lifecy
 
   private JSBundleLoader createDevBundleLoader(String jsFileName, String jsFileSlug) {
     String bundleUrl = bundleUrlForFile(jsFileName);
-    String bundleOut = getReactApplicationContext().getFilesDir().getAbsolutePath() + "/" + jsFileSlug;
+    // nested file directory will not exist in the files dir during development,
+    // so remove any leading directory paths to simply download a flat file into
+    // the root of the files directory.
+    String[] splitFileSlug = jsFileSlug.split("/");
+    String bundleOut = getReactApplicationContext().getFilesDir().getAbsolutePath() + "/" + splitFileSlug[splitFileSlug.length - 1];
 
     Log.d(TAG, "createDevBundleLoader - download web thread to - " + bundleOut);
     downloadScriptToFileSync(bundleUrl, bundleOut);

